### PR TITLE
Weird init behaviour from unittest

### PR DESCRIPTION
**Pull request by gaursagar**

### CODE REVIEW

File: src/cclib/io/cjsonwriter.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def __init__(self, ccdata, terse=False, *args, **kwargs):
         Inputs:
           ccdata - An instance of ccData, parsed from a logfile.
         """
-
+        print ("TERSE:", terse)
```
Comment (gaursagar) [inline src/cclib/io/cjsonwriter.py:32]: Should print `False` when called from the `testcjsonwriter.py` unittest. 
But prints `True`.

This is why unittests did not fail for some files while ccwrite did (for Molpro logs).
Ref: https://github.com/cclib/cclib/issues/377

Comment (TDI-tenderholt) [reply src/cclib/io/cjsonwriter.py:32]: I'm not able to reproduce. I checked out your ```weird-init...``` brach, but I'm seeing ```TERSE: False``` with both Python 2.7.13 and 3.6.1 (both Mac Homebrew in virtual envs).

Comment (TDI-tenderholt) [reply src/cclib/io/cjsonwriter.py:32]: Here's the output for Python3:

```
(Testing) [adam_tenderholt@iMac:~/source/cclib/test/io]$ python3 testcjsonwriter.py 
/Users/adam_tenderholt/Testing/lib/python3.6/site-packages/cclib/parser/logfileparser.py:273: DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() or inspect.getfullargspec()
  if len(inspect.getargspec(self.extract)[0]) != 3:
TERSE: False
.TERSE: False
.
----------------------------------------------------------------------
Ran 2 tests in 0.139s

OK
```

**Comment by gaursagar** [comment]
My bad. I had changed the value elsewhere. Thanks for checking.

**Pull request closed without merging**